registry/application/services/validate_component_service.py

```diff
@@ -0,0 +1,78 @@
+import re
+
+from common import utils
+from common.boto_utils import BotoUtils
+from common.logger import get_logger
+from registry.config import REGION_NAME, COMPONENT_PATH_VALIDATION_PATTERN, COMPONENT_CODE_BUILD_ID
+from registry.infrastructure.repositories.service_publisher_repository import ServicePublisherRepository
+logger = get_logger(__name__)
+
+
+class ValidateDemoComponent:
+    def __init__(self):
+        self.boto_utils = BotoUtils(region_name=REGION_NAME)
+        self.service_repo = ServicePublisherRepository()
+
+    def trigger_demo_component_code_build(self, payload):
+        try:
+            uploaded_file_path = payload["Records"][0]['s3']['object']['key']
+            service_details = self.validate_uploaded_file(uploaded_file_path=uploaded_file_path)
+
+            build_details = {
+                'projectName': COMPONENT_CODE_BUILD_ID,
```

Comment:
Rename COMPONENT_CODE_BUILD_ID to DEMO_COMPONENT_CODE_BUILD_NAME

Reply:
Done